decentraland/unity-renderer
# fix: BiW - Thumbnail is missing in the QuickBar

## What this PR includes?

The thumbnails in the QuickBar disappear when we click on the favorites tab (Fixes #343).

![image.png](https://images.zenhubusercontent.com/60112efc21b0977dcc76d21f/f99f1d30-60a7-4201-aafa-b839052647b3)

## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/BiW-Thumbnail-is-missing-in-the-QuickBar&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=100,100
2. Drag some items from the Catalog to the QuickBar slots.
3. Click on the "Favorites" tab.
4. Notice that all the slots in the QuickBar continue showing the respective items.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
